src/it/resources/com/google/checkstyle/test/chapter7javadoc/rule713atclauses/InputJavaDocTagContinuationIndentation.java

```diff
@@ -43,9 +43,9 @@ class InputJavaDocTagContinuationIndentation implements Serializable
      *     Some javadoc.
      * @return Some text.
      * @serialData Some javadoc.
-     * @deprecated Some text.
-     *    Some javadoc. // warn
      * @throws Exception Some text.
+     *    Some javadoc. // violation '.* incorrect indentation level, expected level should be 4.'
+     * @deprecated Some text.
```

Comment:
Why we changing location of deprecated tag ?
We should not care much on what tag to put violation, let's keep violate deprecated tag

Reply:
It might be due to AtclauseOrder in execution. I got it.
I am ok with such updates.